lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -454,16 +452,32 @@ def update_job_status(self, job_id, status):
         if not (job_id and status):
             raise ValueError("Please provide both job_id and status")
 
-        with self.get_mongo_util().me_collection(self.config["mongo-jobs-collection"]):
-
-            try:
-                job = Job.objects(id=job_id)[0]
-            except Exception:
-                raise ValueError(
-                    "Unable to find job:\nError:\n{}".format(traceback.format_exc())
-                )
+        try:
+            job = self.get_mongo_util().get_job(job_id=job_id)
+        except Exception:
+            raise ValueError(
+                "Unable to find job:\nError:\n{}".format(traceback.format_exc())
```

Comment:
I think it returns a None if it doesn't find a job.

Reply:
@bio-boris yea. But only when you supply valid ObjectID. Otherwise,`mongoengine.errors.ValidationError: 'aaaa' is not a valid ObjectId, it must be a 12-byte input or a 24-character hex string`

Reply:
Good point. You might want to throw an error if the job is None though.

Reply:
👍